src/ripple/rpc/impl/KeypairForSignature.h

```diff
@@ -33,6 +34,7 @@ struct KeyPair
 };
 
 KeyPair keypairForSignature (Json::Value const& params, Json::Value& error);
```

Comment:
Callers still need to know what members the type has — `auto` doesn't save you from having to know whether to write `.first` or `.publicKey`… or was it `.secretKey`? ;-)